### ISSUE TITLE
Train object detection network

### ISSUE THREAD

**Issue by jonathan-hellwig** [HULKs/hulk#120]
For this issue we plan to train a single shot detector (SSD) network to obtain bounding boxes for robots in camera images.  This issue depends on #119.

**Comment by jonathan-hellwig** [comment]
We decided to prioritize other issues for GORE 2023. Now, that the event is done, I have decided to start up the work on robot detection again. I think next up we should focus on the following tasks
- verify the implementation of existing tracking metrics
- implement tracking of additional object detection metrics
- evaluate methods of experiment tracking tools
- conduct a hyperparameter optimization
- explore the impact of different neural network architectures
- explore the impact of different loss functions

**Comment by jonathan-hellwig** [comment]
The current implementation of the models and algorithms can be found at https://github.com/jonathan-hellwig/robot_detection.

**Comment by schluis** [comment]
Tracked further by #638
